dashboard/README.md

```diff
@@ -0,0 +1 @@
+# PLGD Dashboard
```

Comment:
Please add to the readme for now documentation on how to build the dashboard, where should be localization files placed, what format and filename should be used to have them properly loaded.

Reply:
Added readme with basic documentation for build and also for exporting/importing translations.